tests/sentry/integrations/github/test_integration.py

```diff
@@ -585,7 +586,7 @@ def test_github_prevent_install_until_pending_deletion_is_complete(self):
             organization=self.organization, integration=integration
         ).exists()
 
-    def set_rate_limit(self, remaining, limit=5000):
+    def set_rate_limit(self, remaining=MINIMUM_REQUESTS + 100, limit=5000):
```

Comment:
Now calling `set_rate_limit` starts with a high enough value and makes calling the function easier to think about it.